docs/async/async-overview.md

```diff
@@ -1,26 +1,26 @@
 # Async Overview
 
-Modern apps are expected to be responsive and modern services are expected to be elastic:
+Not so long ago, apps got faster simply by buying a newer PC or server and then that trend stopped. In fact, it reversed. Mobile phones appeared with 1ghz single core ARM chips and server workloads transitioned to VMs. Users still want responsive UI and business owners want servers that scale with their business. The transition to mobile and cloud and an internet-connected population of >3B users has resulted in a new set of software patterns. 
 
-*   Client applications are expected to be always-on and always-connected, but also keep the UI free for the user to interact with.
-*   Services are expected to handle spikes in traffic by gracefully scaling up and down.
+* Client applications are expected to be always-on, always-connected and constantly responsive to user interaction (e.g. touch) with high app store ratings!
```

Comment:
'with high app store ratings' feels wrong there. Being always on, always connect, and responsive would result in high app store ratings.